tests/baselines/reference/plainJSGrammarErrors.errors.txt

```diff
@@ -462,8 +460,6 @@ plainJSGrammarErrors.js(205,36): error TS1325: Argument of dynamic import cannot
         outer: for(;;) {
             function test() {
                 break outer
-                ~~~~~~~~~~~
-!!! error TS1107: Jump target cannot cross function boundary.

```

Comment:
Seems odd that this is going away when it's still wrong, right?

Reply:
> Seems odd that this is going away when it's still wrong, right?

Actually i don't get why it's gone you're right there should be that error. As of my understanding this files describes the errors thrown by a specific test and there are assertions made, that when the test ran it should throw these errors. 
When i run the code manually the error is definetly there as it should be but somehow not in this test... 

Is there something I miss?

Reply:
@jakebailey do you have any ideas?

Reply:
I have no idea. Maybe the new `break` in your added code which is making some code behave differently?